src/Pages/SignUp/SignUp.js

```diff
@@ -0,0 +1,93 @@
+import React  from 'react';
+import './SignUp.css'; 
+import fire from '../../Components/firebase';
+import history from '../../Components/history';
+
+class SignUp extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            email:'',
+            password:'',
+            Name:'',
+            Confirm_password:'',
+        }
+        this.submitLogin = this.submitLogin.bind(this);
+    }
+
+
+    submitLogin(e){
+        if( !(this.state.password === this.state.Confirm_password) ){
```

Comment:
user arrow function, so that you don't have to bind 'this'.

example: foo = (...arguments) => { }

Reply:
Ok i will use that..